Helper/Order.php

```diff
@@ -1666,7 +1666,7 @@ public function updateOrderPayment($order, $transaction = null, $reference = nul
     private function createOrderInvoice($order, $transactionId, $amount)
     {
         try {
-            if ($order->getTotalInvoiced() + $amount == $order->getGrandTotal()) {
+            if (round(($order->getTotalInvoiced() + $amount), 2) == round($order->getGrandTotal(), 2)) {
```

Comment:
Why not strict comparison?
`
if (round(($order->getTotalInvoiced() + $amount), 2) === round($order->getGrandTotal(), 2))
`